Ginger/Ginger/Run/RunSetActions/RunSetActionHTMLReportSendEmailEditPage.xaml.cs

```diff
@@ -54,22 +54,18 @@ public RunSetActionHTMLReportSendEmailEditPage(RunSetActionHTMLReportSendEmail r
             {
                 runSetActionHTMLReportSendEmail.Email = new Email();
             }
-            MailFromTextBox.Init(null, runSetActionHTMLReportSendEmail, nameof(RunSetActionHTMLReportSendEmail.MailFrom));
-            MailToTextBox.Init(null, runSetActionHTMLReportSendEmail, nameof(RunSetActionHTMLReportSendEmail.MailTo));
-            MailCCTextBox.Init(null, runSetActionHTMLReportSendEmail, nameof(RunSetActionHTMLReportSendEmail.MailCC));
-            SubjectTextBox.Init(null, runSetActionHTMLReportSendEmail, nameof(RunSetActionHTMLReportSendEmail.Subject));
 
-            BodyTextBox.Init(null, runSetActionHTMLReportSendEmail, nameof(RunSetActionHTMLReportSendEmail.Bodytext));
-            CommentTextBox.Init(null, runSetActionHTMLReportSendEmail, nameof(RunSetActionHTMLReportSendEmail.Comments));
+            RunsetActionDeliveryMethodConfigPageFrame.Content = new RunSetActionDeliveryMethodConfigPage(runSetActionHTMLReportSendEmail.Email);
```

Comment:
Create a object of context once. And use that object for all the bindings instead of duplicating same code of all the fields bindings